lib/index.js

```diff
@@ -54,6 +54,8 @@ SaaSquatch.prototype.identify = function(identify) {
   var paymentProviderId = identify.proxy('traits.paymentProviderId');
   var accountStatus = identify.proxy('traits.accountStatus');
   var referralCode = identify.proxy('traits.referralCode');
+  var userReferralCode = identify.proxy('traits.userReferralCode');
```

Comment:
Should e send all the traits along as is to the tool, mapping any special ones that are match our and their spec?